mne/viz/utils.py

```diff
@@ -2679,7 +2679,9 @@ def _plot_masked_image(ax, data, times, mask=None, picks=None, yvals=None,
     else:
         # imshow for linear because the y ticks are nicer
         # and the masked areas look better
-        extent = [times[0], times[-1], yvals[0], yvals[-1] + 1]
+        top_bound = yvals[-1] + (yvals[-1] - yvals[-2])
```

Comment:
This will break if there are not at least two entries in `yvals` (and same below for `times`), the old code would not.

Granted, the old code would throw a `mpl` warning about not being able to set the bounds properly (or at least `set_xlim` and `set_ylim` do), but this code will throw an error, and it probably shouldn't. It should be acceptable to show an image with a single row (or col).

Reply:
Ok, sure.